src/meson.build

```diff
@@ -0,0 +1,196 @@
+###### xdg-desktop-portal
+
+built_src_portal = gnome.gdbus_codegen('xdp-dbus',
```

Comment:
I appreciate this is essentially a matter of taste, but I would very much prefer if you could suffix variables instead of prefixing them. E.g. `portal_built_sources`, `impl_built_sources`, `xdg_desktop_portal_src`, etc